.travis.yml

```diff
@@ -102,12 +101,12 @@ matrix:
       dist: xenial
       env:
         - USE_PRE=1
-        - DEPENDS="$DEPENDS scipy"
+        - DEPENDS="$DEPENDS scipy statsmodels pandas scikit_learn"
```

Comment:
IIUC, this is that we test the pre-releases of all of these?

Looks good.

Reply:
Yes, and `numpy` too of course